vizro-core/docs/pages/user-guides/run.md

```diff
@@ -80,10 +77,9 @@ The dashboard application can be launched in a Jupyter environment in `inline`,
 - you can specify `jupyter_mode="external"` and a link will be displayed to direct you to the localhost where the dashboard is running.
 - you can use tab mode by `jupyter_mode="tab"` to automatically open the app in a new browser
 
-??? info "Reloading and debugging"
+### Reloading and debugging
```

Comment:
reverted

Reply:
Have we mentioned `Vizro._reset`?

Reply:
Didn't @antonymilne say yesterday that there is no need to add the `Vizro._reset` to the docs? 

Because there are clear error messages where it's already stated and this is currently still a private method no?

Reply:
I think it may be best to put the requirement to mention it into a separate issue & PR because this is more about fixing up admonitions than extending docs (although it is a good opportunity!). Let me ping @antonymilne to see if he can take a look here and give us a steer, and then I'll make the issue if needed and merge this PR.

Reply:
I think `Vizro._reset` is worth a mention here because at the moment it's one of our two officially recommended solutions for running in Jupyter (the other being to restart the kernel). Even though I don't like it and hope it's a temporary solution, it's not going to disappear very soon.

I don't want to talk about it lots and lots in docs for that reason 😅 e.g. in the context of test fixtures as we discussed yesterday, I don't think is worth mentioning. But here I think it's useful.

Reply:
Thank you! And thanks for the suggestion: I'll commit it.